Blp

blazaropinto opened the pull request.

blazaropinto committed:
First update may 2023

typhainepl commented on docs/index.rst:
```diff
@@ -46,4 +46,4 @@ If you use Pfam in your work, please consider citing the :ref:`citing-pfam:Pfam
 ============
 
-If you have any questions or feedback, email us at pfam-help@ebi.ac.uk.
+If you have any questions or feedback, email us at interhelp@ebi.ac.uk.
```
We don't provide the email address directly, please replace the second part of the sentence by "contact us through the `Pfam helpdesk <https://www.ebi.ac.uk/support/interpro>`_"

typhainepl commented on docs/training.rst:
```diff
@@ -8,7 +8,7 @@ Training materials
 ===============
 
-* `Quick tour <https://www.ebi.ac.uk/training-beta/online/courses/pfam-quick-tour>`_ provides a brief introduction to the Pfam website and database.
+* `Quick tour <https://www.ebi.ac.uk/training-beta/online/courses/pfam-quick-tour>`_ provides a brief introduction to the Pfam database and the access to its data through the InterPro website.
```
replace `and the access to its data ` by `and how to access its annotations`...

typhainepl commented on docs/wikipedia.rst:
```diff
@@ -4,43 +4,33 @@
-.. figure:: images/pfam_tab_small.png
-    :width: 200
-    :align: center
+    The **Overview** tab of the Pfam family page shows the associated Wikipedia article if available.
```
replace `family` by `entry`, `shows` by `displays` and `if` by `when`

typhainepl commented on docs/wikipedia.rst:
```diff
@@ -4,43 +4,33 @@
-
 One of the advantages of using Wikipedia to provide our annotations is that any user can now contribute to that annotation text. In many cases, families that do not yet have a Wikipedia article can be assigned an article that already exists. In some cases, however, no suitable article exists, and in that case we would encourage you to consider adding one to Wikipedia yourself.
 
+You can now contribute to the improvement of Pfam annotations in several ways. Besides giving feedbact directly to the curators to improve the traditional description, you can improve existing Wikipedia articles linked to Pfam families. In addition, if you come across a family that does not yet have a Wikipedia article assigned to it, we would really like to add one. If you know of an article tha ...
```
typo `feedback`
remove the `by email ...`

typhainepl commented on docs/wikipedia.rst:
```diff
@@ -60,7 +50,7 @@ How your contribution will be recorded
 Anyone can edit a Wikipedia entry. You can do this either as a new user or you can register with Wikipedia and log on. When you click on the "Edit Wikipedia article" button, your browser will direct you to the edit page for this entry in Wikipedia. If you are a registered user and currently logged in, your changes will be recorded under your Wikipedia user name. However, if you are not a register ...
 
-If you have problems editing a particular page, contact us at pfam-help@ebi.ac.uk and we will try to help.
+If you have problems editing a particular page, contact us at interhelp@ebi.ac.uk and we will try to help.
```
"contact us through the `Pfam helpdesk <https://www.ebi.ac.uk/support/interpro>`_"

typhainepl commented on docs/wikipedia.rst:
```diff
@@ -70,4 +60,4 @@ Pfam has chosen to link families to Wikipedia articles. In some case we have cre
 ----------
 
-If you have problems editing or experience problems with these pages please contact us at pfam-help@ebi.ac.uk
+If you have problems editing or experience problems with these pages please contact us at interhelp@ebi.ac.uk
```
"contact us through the `Pfam helpdesk <https://www.ebi.ac.uk/support/interpro>`_"

typhainepl commented on docs/images/Pfam_entry.png:
What did you use to make this figure? The blue arrows are a bit messy ad the boxes aren't aligned. There is also a grey cross over the figure

blazaropinto replied:
I explained myself on this in the google doc but guess you didn't see it, no problem I'll copy here:
This “scheme” is just an idea to substitute what was there before… so you can let me know what you think about it (besides image quality, I know it needs improvement on that side for sure no worries, just what do you think about something like this as an idea?)

typhainepl replied:
Sorry I had missed your comment on the google doc.
I am not sure I understand the meaning of the blue arrows

blazaropinto replied:
No worries about that.
So I started replicating the old image, but 'Pfam entry' instead of 'Family', 'Clan/Set' instead of 'Clan' and blue instead of yellow. I'm assuming the arrows represent the flow of the data, with the highlighted in blue (previously yellow) meaning 'where the information from a Pfam entry goes' and the rest 'what is contributing to a Pfam entry' (and exchange of info among themselves). I liked the idea of showing "types of information you can find in Pfam" (so tabs on the Pfam entry pages, basically -apart from clans) with this "flow of data", but maybe that's a bit... "abstract" or ... would it be more clear with the same boxes but just double-sided arrows in the same colour for example?

typhainepl replied:
Ok, that's a bit clearer with the explanation. I think we'll need to add a legend for the arrows.
I think it would make sense to link the Proteome, taxonomy and protein boxes to the clan box as they are also provided.

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
-
-Using the "Jump to" search
-==========================
+Users can navigate to specific Pfam entry/family pages by entering the Pfam identifier or accession number or a keyword that form part of its name via three different **Search boxes**:
```
remove `family`

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
-==========================
+Users can navigate to specific Pfam entry/family pages by entering the Pfam identifier or accession number or a keyword that form part of its name via three different **Search boxes**:
 
+1. When selecting the **Browse + By member database** option, it is located in the header of the results table displayed.
```
replace `it is located` by `the search box is located`.
remove `displayed`

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
 
-.. figure:: images/kwSearch.gif
-    :width: 100
+    Example of the top of an InterPro page. A seach box appears when hovering the mouse next to the magnifying glass at the right; it can be used to search for Pfam information.
```
Replace `Example of the top of an InterPro page. A seach box...` by `On the InterPro website header, a search box...`

Typo: search box

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+    :widths: 250 250
+    :header-rows: 1
+
+    * - Search:
```
Remove `:`

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+    :header-rows: 1
+
+    * - Search:
+      - Find:
```
Remove `:`

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
-    :width: 100
+    Example of the top of an InterPro page. A seach box appears when hovering the mouse next to the magnifying glass at the right; it can be used to search for Pfam information.
+
+These text boxes allow you to go quickly to the relevant page in the InterPro site, by using:
```
That's only valid for the header quick text search, not the other 2.
Change text to `This text box allows you...`

blazaropinto replied:
Right, this needs restructuring.
I thought that was also true for the search box in 'Search + By text' though...  not sure if for everything I mentioned in the table.. maybe we can say "The latter two search boxes also allow you to..." (and list the things beside Pfam ID and keywords that are different from the first search box)?

typhainepl replied:
The search box in browse only filters the entries in the list, then the user has to click on the accession or the name to access the Pfam page.
The `Search + By text` provides an `Exact match` which can be clicked to go to the page, it doesn't directly redirect.

blazaropinto replied:
I see, there is an extra click/redirection in the second case 👌 I'll stick to your first suggestion :)

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+    * - Keywords, free text
+      - List of possible matches
+
+**Pfam clans** are called **Sets** in the InterPro website. If a Pfam entry is included in a clan this information will be displayed in the **Overview** tab, next to Set, below the Pfam short name, with a link to the corresponding clan/set page. Additionally, it is possible to browse through the Pfam clans by selecting **Browse + By Set** in the `InterPro website menu <https://www.ebi.ac.uk/inter ...
```
end of paragraph: `select **Pfam** in the database section` instead of `then **Pfam**`

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+
+Pfam entry organisation
+-----------------------
+In each specific entry page, different tabs with relevant information are available:
```
In each Pfam entry page

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
     :align: left
 
-Every page in the Pfam site includes a search box in the page header. You can use this to find Pfam-A families which match a particular keyword. The search includes several different areas of the Pfam database:
+    Example of a Pfam entry `PF02171 <https://www.ebi.ac.uk/interpro/entry/pfam/PF02171/>`_ page. All the tabs explained below are in the left-hand menu. The tab **Overview** is shown by default.
```
can be found left-hand side. 
is displayed

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+
+* **Overview**
+The entry overview tab is the default display, where the type of Pfam entry, the short name and the clan/set 
+(if the entry belongs to any) are shown on top. Normally a curated description of the entry is displayed below, 
```
at the top
replace `Normally` by `Usually,`

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+* **Overview**
+The entry overview tab is the default display, where the type of Pfam entry, the short name and the clan/set 
+(if the entry belongs to any) are shown on top. Normally a curated description of the entry is displayed below, 
+with the pertinent literature references. 
```
replace `pertinent` by `relevant`

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+(if the entry belongs to any) are shown on top. Normally a curated description of the entry is displayed below, 
+with the pertinent literature references. 
+
+If there is a Wikipedia page for the entry, the most general paragraph and the box with an image of a 
```
the first paragraph

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+new tab by clicking on the title.
+
+* **Proteins**
+A list of the protein sequences included in this entry is displayed in this tab. The list can show:
```
The list of proteins matching this entry...
They view can be customised to show:

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+
+1. all proteins (from the whole  UniProtKB database)
+
+2. only 'Reviewed' (from SwissProt - manually curated) 
```
'Reviewed' proteins

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+
+2. only 'Reviewed' (from SwissProt - manually curated) 
+
+3. only 'Unreviewed" (from TrEMBL - derived from public databases automatically integrated into UniProt)
```
'Unreviewed' proteins

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+
+3. only 'Unreviewed" (from TrEMBL - derived from public databases automatically integrated into UniProt)
+
+From each sequence, links to the corresponding InterPro protein and InterPro species pages are displayed next to a small-size protein viewer where the coordinates of the match can be shown by hovering the mouse.
```
For each protein, the corresponding protein page in InterPro can be accessed by clicking on the protein accession or name. The InterPro taxonomy page can be accessed by clicking on the species name. And a small-size protein viewer displays the location of the Pfam entry in the protein, coordinates of the match can be shown by hovering the mouse over it.

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+From each sequence, links to the corresponding InterPro protein and InterPro species pages are displayed next to a small-size protein viewer where the coordinates of the match can be shown by hovering the mouse.
+
+* **Domain architectures**
+This tab shows the various domain architectures of the protein sequences included in the entry, ordered by the number of times that this architecture is seen. Identifying the different domains present in proteins is crucial to understanding how they function.
```
domains arrangement of the proteins matched by the entry...

is crucial to understand...

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+The protein viewer shows a representative sequence for each domain architecture, with the domain size based on the real length of the domain in the protein. When hovering over a domain, more details are available in a tooltip, including the domain's position. From this page, all related Pfam entry pages can also be accessed by clicking on the Pfam accession at the top of the viewer or on the shor ...
+
+* **Taxonomy**
+This tab shows by default a sunburst chart of all the protein species that carry the proteins included in the Pfam entry.
```
all the species that the proteins matched by the Pfam entry belong to.

blazaropinto replied:
thanks! that was badly redacted in the training too (the 'protein species'), updated now :)

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+* **Taxonomy**
+This tab shows by default a sunburst chart of all the protein species that carry the proteins included in the Pfam entry.
+
+By default, eight individual nodes that are derived from the taxonomic lineage of each sequence, ranging from superkingdom down to species, are displayed. For each node in the species tree there is a separate ring - and each ring is arranged radially, with the superkingdoms at the centre and the species around the outermost ring. The length of each ring is proportional to the number of sequences  ...
```
lineage of each protein...

taxonomy tree...

number of proteins found within each taxon ...

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+
+By default, eight individual nodes that are derived from the taxonomic lineage of each sequence, ranging from superkingdom down to species, are displayed. For each node in the species tree there is a separate ring - and each ring is arranged radially, with the superkingdoms at the centre and the species around the outermost ring. The length of each ring is proportional to the number of sequences  ...
+
+Segments of the sunburst chart are coloured according to their superkingdom, as explained in the Legends section. Mousing over any part of the sunburst chart shows the taxonomic name and level, with both the number of sequences and the number species found at that level.
```
Put `Legends` in bold.

with both the number of sequences and the number species found at that level shown on the right-hand side.

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+
+Segments of the sunburst chart are coloured according to their superkingdom, as explained in the Legends section. Mousing over any part of the sunburst chart shows the taxonomic name and level, with both the number of sequences and the number species found at that level.
+
+These data can also be seen as a table and as a tree. In addition, it is possible to choose to see only data from key species instead. These visualisation options can be chosen from the icon panel above the Sunburst.
```
remove upper case in `Sunburst`

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+These data can also be seen as a table and as a tree. In addition, it is possible to choose to see only data from key species instead. These visualisation options can be chosen from the icon panel above the Sunburst.
+
+* **Proteomes**
+A list of the reference proteomes matched by the entry is displayed in this tab. Each item inthis list shows the Proteome ID (which is a link to the Proteome page in InterPro), the name of the species carrying this proteome and the number of proteins in this proteome that match the entry.
```
typo `inthis list`

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+A list of the reference proteomes matched by the entry is displayed in this tab. Each item inthis list shows the Proteome ID (which is a link to the Proteome page in InterPro), the name of the species carrying this proteome and the number of proteins in this proteome that match the entry.
+
+* **Structures**
+This tab displays a list of all the PDB structures that have been determined for protein sequences in the Pfam entry. In each line of the list you can see the PDB accession, the name of the structure in PDB, and a small-sized protein sequence viewer displaying a comparison between the coverage of this structure and the match of the entry in the protein sequence.
```
determined for the proteins matching the Pfam entry.

For each structure, you can see the PDB...

displaying the location of the Pfam entry in the structure.

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+
+Viewing the structures of domains and proteins helps to understand what their function might be, and how individual residues are arranged in the three-dimensional space. Often, two residues which seem distant along the linear protein sequence can be very close in the folded protein. 
+
+By clicking on a PDB accession, name or small image of the structure, a view of the corresponding InterPro structure page that summarises all of the entries of Pfam and other databases and resources for each chain of the structure will be displayed in a protein sequence viewer. The position of each entry within the overall 3D structure can be visualised by choosing the Pfam entry of interest in t ...
```
Additionally, links to the PDBe website and cross-references to other structural databases are provided in the External links section.

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+By clicking on a PDB accession, name or small image of the structure, a view of the corresponding InterPro structure page that summarises all of the entries of Pfam and other databases and resources for each chain of the structure will be displayed in a protein sequence viewer. The position of each entry within the overall 3D structure can be visualised by choosing the Pfam entry of interest in t ...
+
+* **Signature**
+This tab shows the HMM logo of the Pfam model, visualised using `Skylign <http://www.skylign.org/>`_. HMM logos is one way of visualising profile HMMs. Logos provide a quick overview of the properties of an HMM in a graphical form.
```
HMM logos are one way...

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+The visualisation displays the amino acid conservation for each residue in the model. The rendered area can be dragged to a desired position to navigate large logos. Alternatively, a specific residue number can be written in the **Model column** text box. When selecting a particular residue in the logo, the probabilities of each amino acid are displayed in the bottom part.
+
+* **AlphaFold**
+Many of the proteins found in the Pfam entry may have a predicted structure generated by `AlphaFoldDB <https://alphafold.ebi.ac.uk/>`_. A list of all the predicted structures available in AlphaFoldDB for the proteins belonging to this entry is displayed in this tab. For each protein in the list, its Uniprot accession, name, the species it belongs to, its length, and a button that allows you to sh ...
```
replace `UniProt ID` by `UniProt accession`

InterPro protein page and go to the **Alphafold** tab

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+1. The **seed** alignment shows the multiple sequence alignment used to create the HMM model in Pfam. This is a representative set of sequences of the family and it normally has a relatively short number of protein sequences.
+
+2. The **full** alignment shows all the protein sequences from the Uniprot Reference proteomes that match this model.
+
```
All the alignments can be downloaded by clicking on the **Download** button.

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+
+1. In the first section, you can see details about Pfam curators and Sequence ontology.
+
+2. The second section displays the HMM building command used to generate the HMM profile defining the signature.
```
And offers the possibility to download the HMM profile.

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
 =========================================
 
-Searching a protein sequence against the Pfam library of HMMs will enable you to find out the domain architecture of the protein. If your protein is present in the version of UniProt, NCBI Genpept or the metagenomic sequence set that we used to make the current release of Pfam, we have already calculated its domain architecture. You can access this by entering the sequence accession or ID in the  ...
+Searching a protein sequence against the Pfam library of HMMs will enable you to find out the domain architecture of the protein, and thus what its potential function might be. If your protein is present in the version of UniProt used to make the current release of InterPro, we have already calculated its domain architecture. You can access this by entering the Uniprot sequence identifier in any  ...
```
Add a reference to the Search section at the end of the paragraph. To do so add the following, before the search section title : `.. _search:
`
and add a reference to it in the Search ways: :ref:`search`

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+
+    Selecting **Search + By Sequence** in the InterPro website menu.
+
+2. Provide the FASTA formatted sequence(s) (protein or DNA) of interest by pasting them into the test dialogue box or by importing them from a file
```
we only support protein sequences atm
typo: text
remove `dialogue`

blazaropinto replied:
I think I asked about this when doing the tutorial but not sure... DNA sequences were only accepted when running InterProScan locally? or not there either?
I already changed the text here, this is about web services so it was wrong, but wondering if the corresponding needs update in the tutorial..

typhainepl replied:
The DNA sequences are only supported in the local install of InterProScan indeed.

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
 
-If you have a large number of sequences to search (up to several thousand), you can use our batch upload facility. This allows you to upload a file of your sequences in FASTA format, and we will run them against our HMMs and email the results back to you, usually within 48 hours. We request that you put a maximum of 5000 sequences in each file.
+
+4. Click the Search button
```
Click on the **Search** button

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+
+4. Click the Search button
+
+While the sequence search is running, you can continue to navigate through the website, other browser tabs or applications and will get a pop-up notification when the job has been completed (this requires the browser notifications to be allowed).
```
replace `allowed` by `enabled`

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
 
-Proteome analysis
------------------
+Users can search protein sequences that contain specific Pfam entries in a particular arrangement by clicking **Search + By Domain architecture** in the InterPro website menu. Pfam entries that the proteins should or should not contain can be included or excluded from the domain architecture. The **Order of domain matters** option offers the possibility to arrange the domains in a particular orde ...
```
replace `by clicking` by `by selecting`

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+
+    Select **Results + Your InterProScan Searches** in the InterPro website menu. 
+
+All Pfam signatures - and the InterPro entries where they are integrated - are displayed in the protein sequence viewer. The Pfam signatures and InterPro entries are grouped by type (family, domain, repeat, site). The coloured bars indicate the location of signature matches on the protein sequence. Each matched InterPro entry is displayed on a separate line, with the signatures integrated in it d ...
```
replace `signatures` by `entries` in this paragraph, as nowhere else in this document we refer to signature

blazaropinto replied:
I changed this on this paragraph, but should I rephrase the text about the 'Curation' tab?

typhainepl replied:
Yes you can change replace it by `Pfam entry`

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
+    Select **Results + Your InterProScan Searches** in the InterPro website menu. 
+
+All Pfam signatures - and the InterPro entries where they are integrated - are displayed in the protein sequence viewer. The Pfam signatures and InterPro entries are grouped by type (family, domain, repeat, site). The coloured bars indicate the location of signature matches on the protein sequence. Each matched InterPro entry is displayed on a separate line, with the signatures integrated in it d ...
+
```
add image example of protein sequence viewer

blazaropinto replied:
The results from the example sequence are quite simple, maybe it would be a good idea to look here for an example with several sections in the protein viewer (family, domain,unintegrated..), that search lead me to wonder if we should explain somewhere what PfamN is? Maybe just a very brief text in the 'Pfam entries creation and annotation' section with a link to the corresponding blog post?

typhainepl replied:
Yes we should definitely add a section about Pfam-N. You can mention it in this section, no need to develop too much as the blog post explains in details.

typhainepl replied:
Yes we should definitely add a section about Pfam-N. You can mention it in this section, no need to develop too much as the blog post explains in details.

typhainepl commented on docs/getting-started.rst:
```diff
@@ -4,101 +4,244 @@
-====================
+For each Pfam entry, the HMM model is run against the protein sequences belonging to the UniProt Reference Proteomes. Subsequently, Pfam curators set a statistical cut-off, known as a gathering threshold (GA) for an entry. Sequences failing to make a statistical match above this threshold are not reported as hits. The threshold is usually conservative, so that no known false positives are include ...
+
+Different Pfam entries have annotations providing diverse amounts of information. Many of them have a description created by Pfam curators. Anyone can contribute to this annotation by contacting directly the curators.
```
Anyone can contribute to this annotation by contacting directly the curators through the **Add your annotation** toolbox located on the right-hand side of the **Overview** tab.

typhainepl reviewed (commented):
That's a good start, well done!

I feel like the Getting started section is too long and some information is repeated multiple times (e.g. add annotation and wikipedia), I think that we should divide it in multiple files:
- **Getting started** could contain the Site organisation
- **Searching Pfam** could contain the different types of search
1. Searching a specific Pfam entry
2. Searching a protein sequence against Pfam
3. Searching for proteins with a specific domain organisation
- **Pfam entry page organisation**, add an image illustration for each tab, include Pfam entries creation and annotation
- **Clan page organisation** (missing atm)

typhainepl commented on docs/images/overview.png:
Screenshot needs to be taken again with the new top section formatting

blazaropinto replied:
I did that, but (just saying, not sure if that's the expected behaviour) needed to keep refreshing the page as when I scroll down it goes back to the old style.

typhainepl replied:
The wiki one is up to date, but not this one on the repo

blazaropinto committed:
First corrections

blazaropinto committed:
Corrections

blazaropinto committed:
Corrections .

blazaropinto committed:
Corrections

blazaropinto commented:
Thank you for the comments, I think I addressed all of them, please let me know if you see anything weird, but some specific points:

* the image in the getting started section, hope it is more clear now? should I try and improve that? any suggestions?
* in the clan section, I don't really understand the tab Alignments in the website, so don't know how to explain that :S
* I just added a couple of lines about Pfam-N when explaining how to interpret the protein viewer, adding a link to the blog post, is it ok there?
* legends: a lot of them (in pfam entry organisation and in clan organisation mainly) are not showing up in the `autobuild` for me, could you please have a look at the raw files and let me know if I missed something?
* the references to search, doesn't seem to work either... not sure what did I miss there :S

typhainepl committed:
Update getting-started.rst

typhainepl committed:
line length and typo update

typhainepl commented:
It's looking better :)
I have updated a few files, mainly for typo and formatting (avoid having the lines too long).
The new pages were not showing up on the left-hand side panel, as they needed to be added into the index.rst file

A few comments:
- For the sequence search results, the image refers to a protein page, while I've seen that you have an image of the sequence search results (IPresults.png), please replace it and remove
- You didn't add the figure with the different entry types in the get_started
- The alignment shows the relationship between the different members of the clan. Indeed in the examples you have chosen, a lot of members don't have relationship with each other (as it is highlighted in the graph in the Overview tab)
- I corrected the legends issue, it was because you were using the same link multiple times. I solved it by creating a unique link at the top of the document and referring to it in the text.
- I am not sure what you mean by `the references to search`?

typhainepl committed:
revert changes on this file, not finished with it yet

blazaropinto committed:
Fix typo in index.rst

blazaropinto committed:
Fix reference to search in searching-pfam.rst

blazaropinto committed:
Add results figure after search by sequence

blazaropinto committed:
Add description and image of clan alignments

blazaropinto commented:
Thank you for all the updates! Learning referencing here... I'm on it 😁 

Remaining comments:
* I am not sure what you mean by 'figure with the different entry types in the get_started', in the getting started section my idea was just the scheme as the entry types are already explained in the summary, should I add something about that there too?
* I tried to add a description and image for the clan alignment but if you can please double check...
* I added the figure IPresults.png, but the reason I looked for the other example protein was to explain that in the protein viewer you can also see Pfam unintegrated entries and PfamN matches. The alternative option I see here is taking the sequence of this protein instead of the example protein to explain the whole process (I mean, a sequence that gives results with all the features I want to explain). Please let me know your thoughts. For now I left both as a quick solution and for you to see what I mean.

typhainepl committed:
update file names

typhainepl commented:
- The figure I am referring to is the one I mentioned in the Google doc: I think we could replace the different entry types using the image from the Pfam training e.g. [https://docs.google.com/presentation/d/10Po3L0r2-ssj5ZHdMWGfc1ea35H1MIb9QdNaQuYwSck/edit?usp=sharing](https://www.google.com/url?q=https://docs.google.com/presentation/d/10Po3L0r2-ssj5ZHdMWGfc1ea35H1MIb9QdNaQuYwSck/edit?usp%3Dsharing&sa=D&source=docs&ust=1686312625153204&usg=AOvVaw1GjtoQrOfIf4AA74PzZlgH) slide 90
- The clan alignment description looks good to me, I updated the image link, as it was pointing to the proteome clan tab
- For the sequence search results, you can search with the same protein that you used in the protein page image (A1AA27). Pfam-N won't show up in the results of the sequence search, we can add a new question in the FAQ in the documentation to explain what they are

blazaropinto committed:
Update according to last comments

blazaropinto commented:
Sorry I saw and even replied to your comment in the google doc about this image but totally forgot for some reason 😅 , I updated this and the images of the example of search by sequence.
I also change the size and align of the scheme in getting-started for the style to match the other images (including the legend being aligned left).
The sentence about PfamN I left it as a comment for now.

typhainepl committed:
delete unused images

typhainepl committed:
delete unused images

blazaropinto committed:
Update FAQs, add images and links, fix line length

typhainepl commented on docs/getting-started.rst:
```diff
@@ -1,7 +1,7 @@
 ***************
 Getting Started
 ***************
-
```
unnecessary, since it is at the beginning of the document, you can just use the :doc:`Pfam</getting-started>` reference to point to this section. Plus I don't see it used anywhere in other files

typhainepl commented on docs/searching-pfam.rst:
```diff
@@ -123,6 +123,8 @@ The results of the submitted job are accessible by selecting **Results + Your In
   Select **Results + Your InterProScan Searches** in the InterPro website menu. 
 
+.. _protein-viewer:
+
```
I think we should add a title to this section

blazaropinto replied:
...maybe we can put that in another file, not in searching pfam, and just reference it there? like a section on interpreting the protein viewer somewhere like in getting started or pfam entry organisation?

typhainepl replied:
I think it is fine here, just add title like `Interacting with the protein sequence viewer`

blazaropinto replied:
the issue is, this file is divided in searching by text, seq, and dom architectures, basically.
search by seq is divided in  website and locally. and this paragraph is in the middle :S not sure how to do this

typhainepl replied:
a subtitle is fine, we already have one for the Online and Local search, so you can add a title a level below these.

typhainepl committed:
typo and text update

blazaropinto committed:
Update link following feedback

blazaropinto committed:
Add protein viewer subsection
